main/src/ca/uwaterloo/flix/Main.scala

```diff
@@ -103,6 +103,7 @@ object Main {
       XPerfFrontend = cmdOpts.XPerfFrontend,
       XPerfPar = cmdOpts.XPerfPar,
       XPerfN = cmdOpts.XPerfN,
+      xchaosMonkey = cmdOpts.xchaosMonkey,
```

Comment:
We try to keep the order consistent, and i think putting chaos monkey before iterations sounds fine, but then it has to be that way for all the places (ill add comments)